### ISSUE TITLE
Issue on logout

### ISSUE THREAD

**Issue by atellier2** [magenizr/October_Envbar#5]
I have this issue sometime when i logout : 

```
Call to a member function isSuperUser() on null
~/plugins/magenizr/envbar/models/Settings.php line 145
```

With this quick fix it's OK

```
   private function checkPermissions()
    {
        if (!$this->enabled) {
            return false;
        }

        if ($this->superuser) {
            $backendUser = BackendAuth::getUser();
            if ($backendUser) {
                if (!$backendUser->isSuperUser) {
                    return false;
                }
            }
        }

        if (!$this->getEnv()) {
            return false;
        }

        return true;
    }
```

**Comment by magenizr** [comment]
Hi @atellier2 

I can see the problem, thank you for raising the issue! I'll provide a new version in the next 24 hrs.

Regards,
John

**Comment by magenizr** [comment]
Hi @atellier2 

Your fix has been release in version 1.0.6 and is available on the October Marketplace as well. If you run into any issues, please let us know.

Regards,
John
